src/code/ppc-vm.lisp

```diff
@@ -46,7 +46,8 @@
 ;;; Given a signal context, return the floating point modes word in
 ;;; the same format as returned by FLOATING-POINT-MODES.
 ;;;
-;;; FIXME: surely this must be accessible somewhere under *BSD?
+;;; FIXME: surely this must be accessible somewhere under Darwin?  Or
+;;; under NetBSD?
```

Comment:
In order to future-proof this comment, maybe "surely this must be accessible under some other operating systems?"

Reply:
Good idea.